python/shotgun_data/shotgun_data_retriever.py

```diff
@@ -87,6 +87,7 @@ class ShotgunDataRetriever(QtCore.QObject):
     # thumbnail checks are local disk checks and very fast.  These 
     # are always carried out before any shotgun calls
     _CHECK_THUMB_PRIORITY           = 50
+    _CHECK_ATTACHMENT_PRIORITY      = 50
```

Comment:
Feels like attachment should have slightly higher prio than thumbs. There are fewer attachments than there are thumbs and generally i think in a lot of use cases the attachment is 'required' where a thumbnail is often 'nice to have'.